troi/tools/soundcloud_lookup.py

```diff
@@ -0,0 +1,34 @@
+import requests
+from ratelimit import limits, sleep_and_retry
+
+SOUNDCLOUD_URL = f"https://api.soundcloud.com/"
+CALLS=5
+RATE_LIMIT=1
+
+
+@sleep_and_retry
+@limits(calls=CALLS, period=RATE_LIMIT)
+def get_tracks_from_soundcloud_playlist(developer_token, playlist_id):
+    """ Get tracks from the Apple Music playlist.
+    """
+    headers = {
+        "Authorization": f"Bearer {developer_token}",
+    }
+    response = requests.get(SOUNDCLOUD_URL+f"/playlists/{playlist_id}", headers=headers)
+    if response.status_code == 200:
```

Comment:
The if is redundant, you can do `response.raise_for_status()` first and then the contents of this if and remove the if/else statement.